docs/Getting-Started/Create-bootable-media/bootable-usb-using-linux.md

```diff
@@ -0,0 +1,111 @@
+# Create a Bootable Pop!_OS USB in Linux
+
+Use this guide to create Pop!_OS bootable USB media from Linux:
+
+- Install ISO flashing software.
```

Comment:
Is this supposed to be a table of contents? I'm not sure this is necessary, but if it is, then these bullet points need to link to the sections they're referring to.

Reply:
I would also add the Roman numerals in the list at the beginning.

Reply:
Links sound good. What function do roman numerals serve though?

Reply:
I was attempting to mimic this standard section from RHEL documentation. I think including this as a standard section for procedures helps the reader determine if the content is relevant to their needs and technical proficiency. Novice users will want to reference the entire procedure in chronological order, but an advanced user may choose to access content more selectively. Links seem like an obvious improvement. 
![image](https://user-images.githubusercontent.com/80931038/161200027-54198f13-9908-442e-8a8d-4183e4b95320.png)